packages/manager/modules/pci/src/projects/project/ai/apps/dashboard/general-information/update-scaling/update-scaling.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.ai.apps.dashboard.general-information.update-scaling',
+    {
+      url: '/update-scaling',
+      // params: { app: null },
```

Comment:
**nitpick:** Can you remove the comment if params are not needed?

Reply:
https://github.com/ovh/manager/pull/8001/commits/33a7214d2b0990fa80eb54768cc5b204824233b1